scrapscript_tests.py

```diff
@@ -4051,5 +4051,47 @@ def test_pretty_print_variant(self) -> None:
         self.assertEqual(pretty(obj), "#x (a -> b)")
 
 
+class ServerCommandTests(unittest.TestCase):
+    def setUp(self) -> None:
+        import threading, time, os, socket, argparse
+        from scrapscript import server_command
+
+        # Find a random available port
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+            s.bind(("127.0.0.1", 0))
+            self.port = s.getsockname()[1]
+
+        args = argparse.Namespace(
+            directory=os.path.abspath(os.path.join(os.path.dirname(__file__), "examples")),
+            host="127.0.0.1",
+            port=self.port,
+        )
+
+        self.server_thread = threading.Thread(target=server_command, args=(args,))
+        self.server_thread.daemon = True
+        self.server_thread.start()
+
+        time.sleep(0.1)
```

Comment:
This is really sketchy. I think you can/should instead wait on it being ready some other way if possible